modules/trusted-firmware-m/Kconfig

```diff
@@ -58,15 +58,59 @@ config TFM_EXPERIMENTAL
 	select EXPERIMENTAL
 	default y if !TFM_PROFILE_TYPE_MINIMAL
 
+if TFM_PARTITION_CRYPTO
+
 config TFM_CRYPTO_ENGINE_BUF_SIZE
 	int
-	prompt "TFM Crypto engine buffer size" if !TFM_PROFILE_TYPE_MINIMAL
+	prompt "TF-M Crypto - Engine buffer size" if !TFM_PROFILE_TYPE_MINIMAL
 	default 1 if TFM_PROFILE_TYPE_MINIMAL
 	default 12288 if PSA_CRYPTO_DRIVER_ALG_RSA_OAEP_CC3XX || \
 		PSA_CRYPTO_DRIVER_ALG_RSA_PKCS1V15_CRYPT_CC3XX || \
 		PSA_CRYPTO_DRIVER_ALG_RSA_PKCS1V15_SIGN_CC3XX || \
 		PSA_CRYPTO_DRIVER_ALG_RSA_PSS_CC3XX
 	default 8320
+	help
+	  Buffer used by Mbed Crypto for its own allocations at runtime.
+	  This is a buffer allocated in static memory.
+	  To be configured based on the desired use case and application requirements.
+
+config TFM_CRYPTO_CONC_OPER_NUM
+	int
+	prompt "TF-M Crypto - Number of concurrent operations" if !TFM_PROFILE_TYPE_MINIMAL
```

Comment:
No range?

What happens if I put 1_000_000 in here?
Do I run out of memory, or does something only support up to X operations.

I'd suggest:

range 1 1 if TFM_PROFILE_TYPE_MINIMAL
range 1 256

default 1 if TFM_PROFILE_TYPE_MINIMAL
default 8

or a smaller number than 265 if there exists some upper limit that breaks before we run out of memory.

Reply:
this feedback can be dismissed if the author likes it as-is.

Reply:
No range is OK.